docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+services:
+  backup-service:
+    container_name: backup-service
+#    restart: always
+    build: .
+    ports:
+      - "8000:8000"
+    links:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo
+    volumes:
+      - ./data:/data/db
```

Comment:
Please use named volumes unless absolutely necessary (don't think it is here). It is less trouble moving things around then, in case a need arises